tests/proxysql-admin-testsuite.bats

```diff
@@ -169,6 +169,13 @@ fi
   [ "${lines[7]}" = "You have selected No. Terminating." ]
 }
 
+@test "run the check for --force ($WSREP_CLUSTER_NAME)" {
```

Comment:
Need to add more tests, just because the command reported success does not mean that it worked correctly.  Need to verify the functionality (plus ensure code coverage of the various codepaths).
- Need to verify that force works normally (no preexisting servers/users)
- Need to verify that it works in the presence of other servers.
- Also needs to verify that everything still works if --force is not specified.

Reply:
Updated testsuite. Please check